extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -381,6 +381,19 @@ export class Bind {
    * @return {!Promise}
    */
   scanAndApply(addedElements, removedElements, timeout = 2000) {
+    return this.initializePromise_.then(() => {
+      return this.rescan_(addedElements, removedElements, timeout);
```

Comment:
Do these have to run sequentially? Can we just apply the latest rescan?

Reply:
Yea, thought about that e.g. skip elements with class `i-amphtml-binding` during tree walk. 

However, initialization also looks for and sets up amp-bind-macro which is needed if the rescanned contents references it. So we'd need to extract that first. 

Added a TODO so we can revisit later.